source/about-mongosync.txt

```diff
@@ -157,6 +157,32 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+You can periodically run ``mongosync`` against source and destination clusters 
```

Comment:
Thanks @rohshar . What are the steps that a user needs to take in order to continuously run mongosync?

Reply:
They just need to have mongosync run without interruption. 

In practice, this likely means that they run mongosync and then monitor/alert if it goes down. If it goes down then they can start mongosync again and depending on the error it'll be able to catch back up. There are some errors that mongosync can't recover from.

Reply:
Thanks for the explanation. For now, I'll remove that first sentence since it seems it's technically inaccurate and just introduce the section with the "Consider the following..." sentence.